pages/dkp/kommander/2.1/release-notes/2.1.3/index.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: layout.pug
+navigationTitle: Kommander 2.1.3 Release Notes
+title: Kommander 2.1.3 Release Notes
+menuWeight: 40
+excerpt: View release-specific information for Kommander 2.1.3
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Kommander&reg; version 2.1.3 was released on August, 25 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+To get started with Kommander, [download](../../download/) and [install](../../install/) the latest version of Kommander.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Kommander.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Kommander.
+
+## Fixes and Improvements
+
+### Upgrade Konvoy 1.8 with Multiple Availability Zones (Incident 9448)
+
+With this patch release, multiple Availability Zones are now supported when upgrading from Konvoy v1.8.
+
+## Component updates
+
+The following services and service components are upgraded to the listed version:
+
+- centralized-grafana: 18.1.3
+- centralized-kubecost: 0.20.0
+- cert-manager: 0.2.7
+- dex: 2.9.10
+- external-dns: 2.20.5
+- fluent-bit: 0.16.2
+- gatekeeper: 0.6.9
+- grafana-logging: 6.16.14
+- grafana-loki: 0.33.1
+- istio: 1.9.2
+- jaeger: 2.21.0
+- karma: 2.0.0
+- kiali: 1.29.1
+- knative: 0.18.3
+- kube-oidc-proxy: 0.2.5
+- kube-prometheus-stack: 18.1.3
+- kubecost: 0.20.0
+- kubefed: 0.9.0
+- kubernetes-dashboard: 5.0.2
+- kubetunnel: 0.0.8
+- logging-operator: 3.15.0
+- metallb: 0.12.2
+- minio-operator: 4.1.7
+- nfs-server-provisioner: 0.6.0
+- nvidia: 0.4.3
+- project-grafana-logging: 6.16.14
+- project-grafana-loki: 0.33.1
+- project-logging: 1.0.0
+- prometheus-adapter: 2.11.1
+- reloader: 0.0.99
+- thanos: 0.4.5
+- traefik: 10.3.0
+- traefik-forward-auth: 0.3.2
+- velero: 3.1.5
+
+## Known Issues
+
+The following items are known issues with this release.
+
+### Create cert-manager resources on attached clusters with cert-manager pre-installed
```

Comment:
@cwyl02 Do we still need this section? I think we added those missing manifests in 2.1.2?

Reply:
@cwyl02 Following up on @gracedo question, do we need to remove this workaround?